src/index.d.ts

```diff
@@ -4000,6 +4000,146 @@ declare namespace admin.messaging {
      * ID specified in the app manifest.
      */
     channelId?: string;
+
+    /**
+     * Sets the "ticker" text, which is sent to accessibility services. Prior to 
+     * API level 21 (Lollipop), sets the text that is displayed in the status bar 
+     * when the notification first arrives.
+     */
+    ticker?: string;
+
+    /**
+     * When set to `false` or unset, the notification is automatically dismissed when 
+     * the user clicks it in the panel. When set to `true`, the notification persists 
+     * even when the user clicks it.
+     */
+    sticky?: boolean;
+
+    /**
+     * Set the time that the event in the notification occurred. Notifications in the 
```

Comment:
I was a little confused about what kind of "event" this was, and suggested some extra verbiage.  Check out the parallel property in the Java PR and see what you think. :)